test/fixtures/rippled/account-lines.js

```diff
@@ -321,3 +321,37 @@ module.exports.counterparty = function(request, options = {}) {
     }
   });
 };
+
+module.exports.manyItems = function(request, options = {}) {
+  _.defaults(options, {
+    ledger: BASE_LEDGER_INDEX
+  });
+
+  options.marker = 'MORE_TO_COME';
+
+  let lines = [];
```

Comment:
Just a nit: let me know how you feel about this more functional approach:
Outside the `exports`, create a new method:
```
getLinesFromRequest(request) {
  const linesCount = Math.max(request.limit, 10);
  const trustLinesFixture = {
    account: 'rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B',
    balance: '0.3488146605801446',
    ...
  };
  
  return new Array(linesCount).fill(trustLinesFixture);
}
```

cont'd below